.github/goreleaser.yaml

```diff
@@ -0,0 +1,181 @@
+#     test: |
+#       system "#{bin}/pomerium --version"
+
+dockers:
+  - image_templates:
+      - "pomerium/pomerium:{{ .Tag }}"
+      - "pomerium/pomerium:latest"
+    dockerfile: .github/Dockerfile-release
+    binaries:
+      - pomerium
+      - pomerium-cli
+    build_flag_templates:
+      - "--pull"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.name={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+      - "--label=org.opencontainers.image.source={{.GitURL}}"
+      - "--label=repository=http://github.com/pomerium/pomerium"
+      - "--label=homepage=http://www.pomerium.io"
+
+  - goarch: arm64
+    image_templates:
+      - "pomerium/pomerium:arm64v8-{{ .Tag }}"
+      - "pomerium/pomerium:arm64v8-latest"
+    dockerfile: .github/Dockerfile-release.arm64v8
+    binaries:
+      - pomerium
+      - pomerium-cli
+    build_flag_templates:
+      - "--pull"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.name={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+      - "--label=org.opencontainers.image.source={{.GitURL}}"
+      - "--label=repository=http://github.com/pomerium/pomerium"
+      - "--label=homepage=http://www.pomerium.io"
+
+  - goarch: arm
+    goarm: 7
+    image_templates:
+      - "pomerium/pomerium:arm32v7-{{ .Tag }}"
+      - "pomerium/pomerium:arm32v7-latest"
+    dockerfile: .github/Dockerfile-release.arm32v7
+    binaries:
+      - pomerium
+      - pomerium-cli
+
+    build_flag_templates:
+      - "--pull"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.name={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+      - "--label=org.opencontainers.image.source={{.GitURL}}"
+      - "--label=repository=http://github.com/pomerium/pomerium"
+      - "--label=homepage=http://www.pomerium.io"
+
+  - goarch: arm
+    goarm: 6
+    image_templates:
+      - "pomerium/pomerium:arm32v6-{{ .Tag }}"
+      - "pomerium/pomerium:arm32v6-latest"
+    dockerfile: .github/Dockerfile-release.arm32v6
+    binaries:
+      - pomerium
+      - pomerium-cli
+    build_flag_templates:
+      - "--pull"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.name={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+      - "--label=org.opencontainers.image.source={{.GitURL}}"
+      - "--label=repository=http://github.com/pomerium/pomerium"
+      - "--label=homepage=http://www.pomerium.io"
+# changelog:
```

Comment:
Can probably remove unless we want to replace  [github releaser](https://github.com/pomerium/pomerium/blob/master/.github/release-drafter.yml) with go-releaser's built in change log

Reply:
✔️